src/stores/Binding/Store.ts

```diff
@@ -374,6 +374,20 @@ const getInitialState = (
                         ) {
                             state.backfilledBindings.push(UUID);
                         }
+
+                        // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
+                        if (state.collectionMetadata?.[collection]) {
+                            state.collectionMetadata[
+                                collection
+                            ].previouslyBound = liveBindingIndex > -1;
```

Comment:
This condition only exists as a safeguard in the event this action is called outside of hydration.